test/pytorch/test_transforms.py

```diff
@@ -59,5 +59,5 @@ def test_invert_colorize(rgb_min):
 
     input_t = torch.full((8, 3, 32, 32), 255, dtype=torch.uint8)
     out = transfo(input_t)
-    assert torch.all(out <= int(math.ceil(255 * (1 - rgb_min))))
+    assert torch.all(out <= int(1 + math.ceil(255 * (1 - rgb_min))))
```

Comment:
Mmmh, but that's way less accurate, don't you think we should add something like `1e-4`?

Reply:
This is still an issue with 1e-4, because we have a gap of 1 between max(out) and `int(math.ceil(255 * (1 - rgb_min)))`

Reply:
Mmmh, but if it's not a rounding error, I'm not sure it's supposed to happen :thinking: 
Any idea why?

Reply:
I happens only with non-normalized images, I fixed it the same way we did for normalized images